node_modules/oae-principals/tests/test-users.js

```diff
@@ -1839,15 +1810,22 @@ describe('Users', function() {
                                                     // Verify an incorrect email is invalid
                                                     PrincipalsTestUtil.assertUpdateUserFails(jack.restContext, jack.user.id, {'email': 'so incorrect'}, 400, function() {
 
-                                                        // Make sure that the user's basic profile is unchanged
-                                                        RestAPI.User.getUser(jack.restContext, jack.user.id, function(err, userObj) {
-                                                            assert.ok(!err);
-                                                            assert.ok(userObj);
-                                                            assert.equal(userObj.visibility, jack.user.visibility);
-                                                            assert.equal(userObj.displayName, jack.user.displayName);
-                                                            assert.equal(userObj.resourceType, 'user');
-                                                            assert.equal(userObj.profilePath, '/user/' + userObj.tenant.alias + '/' + AuthzUtil.getResourceFromId(userObj.id).resourceId);
-                                                            return callback();
+                                                        // Verify unknown fields aren't allowed
+                                                        PrincipalsTestUtil.assertUpdateUserFails(jack.restContext, jack.user.id, {'wumptiedumpty': true}, 400, function() {
+
+                                                            // Make sure that the user's basic profile is unchanged
+                                                            RestAPI.User.getUser(jack.restContext, jack.user.id, function(err, userObj) {
+                                                                assert.ok(!err);
+                                                                assert.ok(userObj);
+                                                                assert.equal(userObj.visibility, jack.user.visibility);
+                                                                assert.equal(userObj.displayName, jack.user.displayName);
+                                                                assert.equal(userObj.resourceType, 'user');
+                                                                assert.equal(userObj.profilePath, '/user/' + userObj.tenant.alias + '/' + AuthzUtil.getResourceFromId(userObj.id).resourceId);
+                                                                assert.equal(userObj.email, jack.user.email);
+                                                                assert.equal(userObj.emailChange, jack.user.emailChange);
+                                                                assert.equal(userObj.emailVerified, jack.user.emailVerified);
```

Comment:
These assertions should be removed. Can you do a quick `git grep` for these and make sure they're all gone?